### PR TITLE
Update Slovak translation

### PR DESCRIPTION
**Pull request by mvalec**
Complete slovak translation. Missing unused texts.

### CODE REVIEW

File: language/resource.language.sk_sk/strings.po [hunk 1]
```diff
@@ -1,28 +1,41 @@
...
+msgstr "Kompletne prepracovaný dizajn s moderným vzhľadom pri zachovaní klasického štýlu Aeon."
 
 msgctxt "#31000"
 msgid "Directed by"
 msgstr "Režia"
 
 msgctxt "#31001"
 msgid "Written by"
-msgstr "Scenár"
+msgstr "Autor: "
```
Comment (mikesilvo164) [inline language/resource.language.sk_sk/strings.po]: Please remove colon from msgstr

File: language/resource.language.sk_sk/strings.po [hunk 2]
```diff
@@ -286,7 +323,7 @@ msgstr "Presuňte položku menu"
 
 msgctxt "#31074"
 msgid "Choose item to swap places with"
-msgstr "Vyberte položku s ktorou chcete zameniť umiestnenie"
+msgstr "Vyberte položku, s ktorou chcete zameniť umiestnenie"
```
Comment (mikesilvo164) [inline language/resource.language.sk_sk/strings.po]: Is this comma necessary?

File: language/resource.language.sk_sk/strings.po [hunk 3]
```diff
@@ -584,113 +1313,1281 @@ msgctxt "#31321"
...
+msgid "Party mode (Music videos)"
+msgstr "Párty režim (Videoklipy)"
+
+msgctxt "#31520"
+msgid "Movies in set"
+msgstr "Filmy v kolekcii"
+
+msgctxt "#31521"
+msgid "More movies starring"
+msgstr "More movies starring"
```
Comment (mikesilvo164) [inline language/resource.language.sk_sk/strings.po]: Currently not used but may be in the future. Want to translate it now?

File: language/resource.language.sk_sk/strings.po [hunk 4]
```diff
@@ -584,113 +1313,1281 @@ msgctxt "#31321"
...
+msgid "Building Menu"
+msgstr "Vytváram ponuku"
+
+msgctxt "#31527"
+msgid "Please wait"
+msgstr "Čakajte prosím"
+
+msgctxt "#31528"
+msgid "PVR stream"
+msgstr "PVR stream"
```
Comment (mikesilvo164) [inline language/resource.language.sk_sk/strings.po]: Translation needed?

File: language/resource.language.sk_sk/strings.po (hunk 1 above)
Comment (mvalec) [reply language/resource.language.sk_sk/strings.po]: It's my mistake. Original text "Scenár" is right.
```suggestion
msgstr "Scenár"
```

File: language/resource.language.sk_sk/strings.po (hunk 2 above)
Comment (mvalec) [reply language/resource.language.sk_sk/strings.po]: Yes, comma is necessary in Slovak.

File: language/resource.language.sk_sk/strings.po (hunk 4 above)
Comment (mvalec) [reply language/resource.language.sk_sk/strings.po]: Translation is not needed.

File: language/resource.language.sk_sk/strings.po (hunk 3 above)
Comment (mvalec) [reply language/resource.language.sk_sk/strings.po]: Now I'm not quite sure. If it's related to the actor and his cast in other movies, this is the translation:
```suggestion
msgstr "Ďalšie účinkovanie"
```

**Comment by mikesilvo164** [comment]
Looks good!

Thanks 😃

**Pull request merged**